LanternExtractor/EQ/Wld/Exporters/GltfWriter.cs

```diff
@@ -1181,16 +1183,17 @@ public class WldMeshHelper
         private readonly ISet<DataTypes.Polygon> _uniqueTriangles;
         private readonly IDictionary<int, Vector3> _wldVertexIndexToDuplicatedVertexNormals;
         private readonly TriangleVertexSetComparer _triangleSetComparer;
-
+        private readonly Matrix4x4 _transformMatrix;
 		private static readonly Vector4 DefaultVertexColor = new Vector4(0f, 0f, 0f, 1f); // Black
 
-		public WldMeshHelper(Mesh wldMesh, bool separateTwoFacedTriangles)
+		public WldMeshHelper(Mesh wldMesh, bool separateTwoFacedTriangles, bool mirrorAxis = true)
```

Comment:
Minor - mirrorXAxis instead of mirrorAxis

Reply:
Fixed